### ISSUE TITLE
Link wizard missing in TYPO3 10

### ISSUE THREAD

**Issue by fiedomedia** [FluidTYPO3/flux#1780]
Hi,

after upgrading to TYPO3 v10 with latest flux dev-branch the link wizard is missing.
It just displays a simple input field.
My config that worked with TYPO3 v9:
```
<flux:field.input name="link" label="Link">
	<flux:wizard.link activeTab="page" />
</flux:field.input>
```

**Comment by fiedomedia** [comment]
Solution for migration:
Replace old code with
```
<flux:field type="input"
	name="link"
	label="Link"
	config="{renderType: 'inputLink'}" />
```

**Comment by w010** [comment]
> Solution for migration:
> Replace old code with  (...)
Man, that information was really hard to find. I just googled everywhere how link fields are to be done now in 10.4 and nothing. Not in the web, not in the doc, not in the ext / readme / code comments, nothing. Finally I found it here.
Such link wizard is used in almost every flux content, I wonder why that migration is so badly documented, only mentions that it differs now... Weird
